src/modules/FileActionsMenu/FileActionsMenu.Plugins/PowerToys.FileActionsMenu.Plugins.PowerToys/ImageResizer.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+using FileActionsMenu.Helpers;
+using FileActionsMenu.Helpers.Telemetry;
+using FileActionsMenu.Interfaces;
+using FileActionsMenu.Ui.Helpers;
+using Microsoft.PowerToys.Settings.UI.Library;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace PowerToys.FileActionsMenu.Plugins.PowerToys
+{
+    internal sealed class ImageResizer : IAction
+    {
+        private string[]? _selectedItems;
+
+        public string[] SelectedItems { get => _selectedItems.GetOrArgumentNullException(); set => _selectedItems = value; }
+
+        public string Title => SelectedItems.Length == 1 ? ResourceHelper.GetResource("PowerToys.ImageResizer.Title_S") : ResourceHelper.GetResource("PowerToys.ImageResizer.Title_P");
+
+        public IAction.ItemType Type => IAction.ItemType.SingleItem;
+
+        public IAction[]? SubMenuItems => null;
+
+        public int Category => 3;
+
+        public IconElement? Icon => IconHelper.GetIconFromModuleName("ImageResizer");
+
+        public bool IsVisible => SelectedItems.All(path => path.IsImage()) && GPOWrapperProjection.GPOWrapper.GetConfiguredImageResizerEnabledValue() != GPOWrapperProjection.GpoRuleConfigured.Disabled && SettingsRepository<GeneralSettings>.GetInstance(new SettingsUtils()).SettingsConfig.Enabled.ImageResizer;
```

Comment:
This policy check won't work as expected if the module is force enabled by policy. Correct? Then you need to improve the check.

Reply:
Why? As long as it is not disabled (either manually or by policy) it will show (regardless of how it is enabled. Or what am I missing here exactly?

Reply:
The policy can override manual disabling (force enabled). This should happen if policy value equals `enabled`.